test/ProjectService.Business.UnitTests/Commands/DeleteRoleCommandTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.ProjectService.Business.Commands;
+using LT.DigitalOffice.ProjectService.Business.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Business.UnitTests.Commands
+{
+    class DeleteRoleCommandTests
+    {
+        private IDeleteRoleCommand command;
+
+        private Mock<IRoleRepository> repositoryMock;
+        private Mock<IAccessValidator> accessValidatorMock;
+
+        private Guid roleId;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            roleId = Guid.NewGuid();
+        }
+        
+        [SetUp]
+        public void SetUp()
+        {
+            repositoryMock = new Mock<IRoleRepository>();
+            accessValidatorMock = new Mock<IAccessValidator>();
+            command = new DeleteRoleCommand(repositoryMock.Object, accessValidatorMock.Object);
+        }
+
+        [Test]
+        public void ShouldThrowExceptionWhenRepositoryThrowsIt()
+        {
+            accessValidatorMock
+                .Setup(x => x.IsAdmin())
+                .Returns(true);
```

Comment:
wrong mock validator, check this in [](https://github.com/LT-Students/DigitalOffice-UserService/blob/develop/test/UserService.Business.UnitTests/CreateUserCommandTests.cs)